### ISSUE TITLE
Installed srcml problem with rpath finding libsrcml on macOS

### ISSUE THREAD

**Issue by mlcollard** [srcML/srcML#1919]
A problem just crept up. Not sure when it occurred or why I did not notice it earlier.

When you build and install in branch develop on macOS you get the following:

```console
$ srcml --version
dyld[68823]: Symbol not found: _srcml_append_transform_param
  Referenced from: <E83992BE-F577-30AD-9209-A5F8E28599A0> /usr/local/bin/srcml
  Expected in:     <no uuid> unknown
Abort trap: 6
```

The problem is that the executable srcml cannot find the library libsrcml. This is supposed to be handled via the RPATH. The RPATH path in the srcml executable does not have a directory. So, the system should be looking at /usr/local/lib, but it is not. 

Anyhow, there is a fix of this to have the executable append /usr/local/lib to the rpath directories.

BTW, this does **not** occur when running the executable in the bin directory of the build. There, the rpath knows to look in that directory for libsrcml.

**Comment by mlcollard** [comment]
The solution appears to be the following:

```CMake
set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
list(APPEND CMAKE_INSTALL_RPATH "/usr/local/lib")
```

**Comment by mlcollard** [comment]
Fixed in pr #1920
